onnx2kerastl/converter.py

```diff
@@ -155,6 +156,12 @@ def onnx_to_keras(onnx_model, input_names, name_policy=None, verbose=True, chang
                 set_weights_names = set(attached_weights_names)
                 set_weights_names = "__".join(set_weights_names)
                 layer_name = output.replace(":", "_")
+                while layer_name[0] == '/':
+                    layer_name = layer_name.replace("/", "", 1)
+
+                if layer_name == "":
+                    layer_name = str(uuid.uuid4())[:10]
+
```

Comment:
One day we'll need to split this function into a few :P

Reply:
haha sure!